lib/OpenQA/Git.pm

```diff
@@ -81,7 +81,7 @@ sub set_to_latest_master ($self, $args = undef) {
         return $self->_format_git_error($res, 'Unable to reset repository to origin/master') unless $res->{status};
     }
 
-    return undef;
+    return;
```

Comment:
why this change? It's good style to make a return always return a valid type explicitly hence the `return undef`

Reply:
Coming from other programming languages, e.g. python, most of the time is preferred to use the return without explicitly a value. I'll update it with `return undef`.